components/ui/floating-elements.tsx

```
@@ -173,6 +132,7 @@ export function FloatingElements({
 
     // Cleanup function
     return () => {
+      tweens.forEach(t => t.kill())
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)